backend/src/routes/api/status/statusUtils.ts

```diff
@@ -33,6 +34,7 @@ export const status = async (
         clusterBranding,
         isAdmin,
         isAllowed,
+        saToken,
```

Comment:
Why do you need this in the frontend? I didn't see why you needed it in your last PR either...